trust_stores_observatory/store_fetcher/java_fetcher.py

```diff
@@ -0,0 +1,115 @@
+from typing import Tuple, List
+from tempfile import NamedTemporaryFile
+import subprocess
+import re
```

Comment:
I would try to not use regular expressions at all in this file (see my comments below) :)

Reply:
...I can't find them below?? 🔍 😁

Reply:
Sorry - I just meant the other comments (they were displayed below while I was doing the review).